tests/scripts/components-sanitizers.sh

```diff
@@ -142,9 +169,28 @@ component_test_tsan () {
     make test
 }
 
+component_test_tsan () {
```

Comment:
Rebase error: this component was duplicated.


Hint: to review the rebase from 37514bf to be59951:

```
git range-diff origin/mbedtls-3.6 37514bf be59951
```

Reply:
Fixed, sorry about that